users/src/test/java/io/crate/operation/user/TransportPrivilegesActionTest.java

```diff
@@ -22,79 +22,42 @@
     private static final Set<Privilege> PRIVILEGES = new HashSet<>(Arrays.asList(GRANT_DQL, GRANT_DML));
-    private static final List<String> USERNAMES = Arrays.asList("Ford", "Arthur");
-
-    private MetaData.Builder mdBuilder;
-
-    @Before
-    public void setUpDefaultUsersPrivilegesMetaData() {
-        mdBuilder = MetaData.builder();
-        Map<String, Set<Privilege>> usersPrivileges = new HashMap<>();
-        for (String userName : USERNAMES) {
-            usersPrivileges.put(userName, new HashSet<>(PRIVILEGES));
-        }
-        mdBuilder.putCustom(UsersPrivilegesMetaData.TYPE, new UsersPrivilegesMetaData(usersPrivileges));
-    }
-
-    @Test
-    public void testApplyPrivilegesSameExists() throws Exception {
-        PrivilegesRequest request = new PrivilegesRequest(USERNAMES, PRIVILEGES);
-
-        long rowCount = TransportPrivilegesAction.applyPrivileges(mdBuilder, request);
-        assertThat(rowCount, is(0L));
-    }
 
     @Test
-    public void testRevokeWithoutGrant() throws Exception {
-        PrivilegesRequest request = new PrivilegesRequest(Arrays.asList("Arthur"), Arrays.asList(
-            new Privilege(Privilege.State.REVOKE, Privilege.Type.DCL, Privilege.Clazz.CLUSTER, null, "crate")));
-
-        long rowCount = TransportPrivilegesAction.applyPrivileges(mdBuilder, request);
-        assertThat(rowCount, is(0L));
-        UsersPrivilegesMetaData privilegesMetaData = (UsersPrivilegesMetaData) mdBuilder.getCustom(UsersPrivilegesMetaData.TYPE);
-        assertThat(privilegesMetaData.getUserPrivileges("Arthur"), is(PRIVILEGES));
-    }
-
-    @Test
-    public void testRevokeWithGrant() throws Exception {
-        PrivilegesRequest request = new PrivilegesRequest(Arrays.asList("Arthur"), Arrays.asList(
-            new Privilege(Privilege.State.REVOKE, Privilege.Type.DML, Privilege.Clazz.CLUSTER, null, "crate")));
-
-        long rowCount = TransportPrivilegesAction.applyPrivileges(mdBuilder, request);
-        assertThat(rowCount, is(1L));
-        UsersPrivilegesMetaData privilegesMetaData = (UsersPrivilegesMetaData) mdBuilder.getCustom(UsersPrivilegesMetaData.TYPE);
-        assertThat(privilegesMetaData.getUserPrivileges("Arthur"), contains(GRANT_DQL));
-    }
-
-    @Test
-    public void testRevokeWithGrantOfDifferentGrantor() throws Exception {
-        PrivilegesRequest request = new PrivilegesRequest(Arrays.asList("Arthur"), Arrays.asList(
-            new Privilege(Privilege.State.REVOKE, Privilege.Type.DML, Privilege.Clazz.CLUSTER, null, "hoschi")));
-
-        long rowCount = TransportPrivilegesAction.applyPrivileges(mdBuilder, request);
-        assertThat(rowCount, is(1L));
-        UsersPrivilegesMetaData privilegesMetaData = (UsersPrivilegesMetaData) mdBuilder.getCustom(UsersPrivilegesMetaData.TYPE);
-        assertThat(privilegesMetaData.getUserPrivileges("Arthur"), contains(GRANT_DQL));
+    public void testApplyPrivilegesCreatesNewPrivilegesInstance() {
```

Comment:
maybe rename to `testApplyPrivilegesCreatesNewUsersPrivilegesMetaDataInstance` to make it a bit more clear?